tests/JsonQueryWrapper/JsonQueryTest.php

```diff
@@ -73,4 +73,14 @@ public function testSomething()
 
         $this->assertTrue($jq->run('.Foo.Bar == 33'));
     }
+
+    public function testFixProcessBuilderPileup()
+    {
+        $pb = new ProcessBuilder();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $pb. Configured minimum length is 3.](https://www.codacy.com/app/estahn/json-query-wrapper/file/1949678621/issues/source?bid=3105591&fileBranchId=3107888#l79)